CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 0.0.99 - WIP
+
+## Serialization Compatibility
+
+ * Due to a bug discovered in 0.0.98, if a `ChannelManager` is serialized on
+   version 0.0.98 while an `Event::PaymentSent` is pending processing, the
+   `ChannelManager` will fail to deserialize both on version 0.0.98 and later
+   versions. If you have such a `ChannelManager` available, a simple patch will
```

Comment:
What would the patch ? Adding back a `payment.preimage: Readable::read(reader)?`

Reply:
Yea, just reading (and throwing away) an extra 32 bytes.